assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -22,71 +22,92 @@ import placeholderShippingMethods from '../../placeholder-shipping-methods';
-
-	const currency = getCurrencyFromPriceResponse( cartTotals );
-	const totalItems = [
+const getTotalRows = ( cartTotals ) => {
+	const totalItems = parseInt( cartTotals.total_items );
+	const totalItemsTax = parseInt( cartTotals.total_items_tax );
+	const totalRows = [
 		{
 			label: __( 'List items:', 'woo-gutenberg-products-block' ),
 			value: DISPLAY_PRICES_INCLUDING_TAXES
-				? cartTotals.total_items + cartTotals.total_items_tax
-				: cartTotals.total_items,
+				? totalItems + totalItemsTax
+				: totalItems,
 		},
 	];
-	if ( cartTotals.total_fees > 0 ) {
-		totalItems.push( {
+	const totalFees = parseInt( cartTotals.total_fees );
+	if ( totalFees > 0 ) {
+		const totalFeesTax = parseInt( cartTotals.total_fees_tax );
+		totalRows.push( {
 			label: __( 'Fees:', 'woo-gutenberg-products-block' ),
 			value: DISPLAY_PRICES_INCLUDING_TAXES
-				? cartTotals.total_fees + cartTotals.total_fees_tax
-				: cartTotals.total_fees,
+				? totalFees + totalFeesTax
+				: totalFees,
 		} );
 	}
-	if ( cartTotals.total_discount > 0 ) {
-		totalItems.push( {
+	const totalDiscount = parseInt( cartTotals.total_discount );
+	if ( parseInt( totalDiscount ) > 0 ) {
+		const totalDiscountTax = parseInt( cartTotals.total_discount_tax );
+		totalRows.push( {
 			label: __( 'Discount:', 'woo-gutenberg-products-block' ),
 			value: DISPLAY_PRICES_INCLUDING_TAXES
-				? cartTotals.total_discount + cartTotals.total_discount_tax
-				: cartTotals.total_discount,
+				? totalDiscount + totalDiscountTax
+				: totalDiscount,
 		} );
 	}
 	if ( ! DISPLAY_PRICES_INCLUDING_TAXES ) {
-		totalItems.push( {
+		const totalTax = parseInt( cartTotals.total_tax );
+		totalRows.push( {
 			label: __( 'Taxes:', 'woo-gutenberg-products-block' ),
-			value: cartTotals.total_tax,
+			value: totalTax,
 		} );
 	}
-	totalItems.push( {
+	const totalShipping = parseInt( cartTotals.total_shipping );
+	const totalShippingTax = parseInt( cartTotals.total_shipping_tax );
+	totalRows.push( {
 		label: __( 'Shipping:', 'woo-gutenberg-products-block' ),
 		value: DISPLAY_PRICES_INCLUDING_TAXES
-			? cartTotals.total_shipping + cartTotals.total_shipping_tax
-			: cartTotals.total_shipping,
+			? totalShipping + totalShippingTax
+			: totalShipping,
 		description: __(
 			'Shipping to location (change address)',
 			'woo-gutenberg-products-block'
 		),
 	} );
 
+	return totalRows;
+};
+
+/**
+ * Component that renders the Cart block when user has something in cart aka "full".
+ */
+const Cart = () => {
+	// @todo this are placeholders
+	const onActivateCoupon = ( couponCode ) => {
+		// eslint-disable-next-line no-console
+		console.log( 'coupon activated: ' + couponCode );
+	};
+	const cartTotals = {
+		currency: 'EUR',
+		currency_minor_unit: 2,
+		total_items: '6000',
+		total_items_tax: '0',
+		total_fees: '0',
+		total_fees_tax: '0',
+		total_discount: '0',
+		total_discount_tax: '0',
+		total_shipping: '0',
+		total_shipping_tax: '0',
+		total_tax: '0',
+		total_price: '6000',
+	};
```

Comment:
I think we should define this sample content outside of the component otherwise it's going to be a new object instance every time the function component is invoked (which will be a lot because of how react calls functional components).

Reply:
Well, that will go away once this component is hooked up to the API, but moved it out of the component anyway in 3c6c9c0e628cf3cfb70c7c94b8669713e648578c.

I moved `onActivateCoupon` out as well. We can move it inside in the future if it needs to read props or interact with the component state.